yarn-project/circuits.js/src/contract/contract_address.ts

```diff
@@ -15,7 +15,7 @@ import { type ContractInstance } from './interfaces/contract_instance.js';
  * ```
  * salted_initialization_hash = pedersen([salt, initialization_hash, deployer], GENERATOR__SALTED_INITIALIZATION_HASH)
  * partial_address = pedersen([contract_class_id, salted_initialization_hash], GENERATOR__CONTRACT_PARTIAL_ADDRESS_V1)
- * address = poseidon2Hash([public_keys_hash, partial_address, GENERATOR__CONTRACT_ADDRESS_V1])
+ * address = (poseidon2Hash([public_keys_hash, partial_address, GENERATOR__CONTRACT_ADDRESS_V1]) * G) + ivpk_m
```

Comment:
This is missing the `y > half_range` check, and the fact that we only take the x coord